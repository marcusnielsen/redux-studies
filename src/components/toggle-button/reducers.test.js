import RootReducer, {update} from './reducers'

test('update', function onTest () {
  const state = {active: false}
  const eventData = {active: true}
  expect(update(state, eventData)).toEqual({active: true})
})

test('rootReducer', function onTest () {
  const initialState = null
  const rootReducer = RootReducer({initialState})

  expect(Object.keys(rootReducer)).toMatchSnapshot()
})
