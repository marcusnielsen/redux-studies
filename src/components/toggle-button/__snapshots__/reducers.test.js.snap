exports[`test rootReducer 1`] = `Array []`;
